### PR TITLE
Add next-seo example

### PR DESCRIPTION
**Pull request by garmeeh**
This PR adds a basic example using [next-seo](https://www.npmjs.com/package/next-seo).

### CODE REVIEW

File: examples/with-next-seo/package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "with-next-seo",
+  "version": "1.0.0",
+  "scripts": {
+    "dev": "next",
+    "build": "next build",
+    "start": "next start"
+  },
+  "author": "Gary Meehan <garymeehan@outlook.com> (https://www.garymeehan.ie/)",
```
Comment (timneutkens) [inline examples/with-next-seo/package.json]: Could you make the author/license the same as the other examples?

**Review by timneutkens — APPROVED** [review]

**Comment by timneutkens** [comment]
Besides the change to the author the examples looks fine 👍

**Comment by garmeeh** [comment]
Thanks @timneutkens, updated there now.

**Comment by timneutkens** [comment]
Thanks!

**Pull request merged**